### PR TITLE
Fix fallback mechanism 

### PR DESCRIPTION
**Pull request by DmitriyMolch**
1. Stop it after the transfer is finished.
2. Fix multiple fallback mechanism initializations.

https://cerenetwork.atlassian.net/browse/CBI-2234

### CODE REVIEW

File: src/Contexts/Adaptors/EVMAdaptors/EVMDestinationAdaptorProvider.tsx [hunk 1]
```diff
@@ -207,7 +206,7 @@ export const EVMDestinationAdaptorProvider = ({
       Math.max(pollingMinIntervalMs, 3 * blockTimeMs),
       pollingMaxIntervalMs
     );
-    const fallback = new Fallback(delayMs, pollingIntervalMs, async () => {
+    const flb = new Fallback(delayMs, pollingIntervalMs, async () => {
```
Comment (AndreiNavoichyk) [inline src/Contexts/Adaptors/EVMAdaptors/EVMDestinationAdaptorProvider.tsx]: Please, rename back to `fallback`

File: src/Contexts/Adaptors/SubstrateDestinationAdaptor.tsx [hunk 2]
```diff
@@ -153,7 +159,7 @@ export const SubstrateDestinationAdaptorProvider = ({
       Math.max(pollingMinIntervalMs, 3 * blockTimeMs),
       pollingMaxIntervalMs
     );
-    const fallback = new Fallback(delayMs, pollingIntervalMs, async () => {
+    const flb = new Fallback(delayMs, pollingIntervalMs, async () => {
```
Comment (AndreiNavoichyk) [inline src/Contexts/Adaptors/SubstrateDestinationAdaptor.tsx]: Please, rename back to `fallback`

File: src/Utils/Fallback.tsx [hunk 3]
```diff
@@ -9,6 +9,7 @@ export class Fallback {
   ) {}
 
   start(): void {
+    if(this.started()) return;
```
Comment (AndreiNavoichyk) [inline src/Utils/Fallback.tsx]: Add ` ` after `if`

File: src/Contexts/Adaptors/EVMAdaptors/EVMDestinationAdaptorProvider.tsx (hunk 1 above)
Comment (DmitriyMolch) [reply src/Contexts/Adaptors/EVMAdaptors/EVMDestinationAdaptorProvider.tsx]: Fixed

File: src/Contexts/Adaptors/SubstrateDestinationAdaptor.tsx (hunk 2 above)
Comment (DmitriyMolch) [reply src/Contexts/Adaptors/SubstrateDestinationAdaptor.tsx]: Fixed

File: src/Utils/Fallback.tsx (hunk 3 above)
Comment (DmitriyMolch) [reply src/Utils/Fallback.tsx]: Fixed

**Review by AndreiNavoichyk — APPROVED** [review]

**Pull request merged**